wagtail/admin/tests/pages/test_dashboard.py

```diff
@@ -111,11 +113,36 @@ def test_panel(self):
 
         page = Page.objects.get(pk=self.child_page.id).specific
 
-        # check if the revision is the revision of edited Page
-        self.assertEqual(ctx["last_edits"][0][0].content_object, page)
-        # check if the page in this list is the specific page of this revision
+        # check the timestamp matches the edit
+        self.assertEqual(ctx["last_edits"][0][0], edit_timestamp)
+        # check if the page in this list is the specific page
         self.assertEqual(ctx["last_edits"][0][1], page)
 
+    def test_copying_does_not_count_as_an_edit(self):
+        self.login(username="bob", password="password")
+        # change a page
+        self.change_something("Bob's edit")
+
+        # copy the page
+        post_data = {
+            "new_title": "Goodbye world!",
+            "new_slug": "goodbye-world",
+            "new_parent_page": str(self.root_page.id),
+            "copy_subpages": False,
+            "alias": False,
+        }
+        self.client.post(
+            reverse("wagtailadmin_pages:copy", args=(self.child_page.id,)), post_data
+        )
+        # check that page has been copied
+        self.assertTrue(Page.objects.get(title="Goodbye world!"))
+
+        response = self.client.get(reverse("wagtailadmin_home"))
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, "Your most recent edits")
+        self.assertNotContains(response, "Hello world!")
```

Comment:
Hold on, that was meant to be an assertContains, but it accidentally passes because we changed the title to "Bob's edit" :-) Fixing...